.gitlab-ci.yml

```diff
@@ -8,6 +8,8 @@ variables:
 build-docker-image-amd64:
   stage: build
   only: [ "tags" ]
+  when: manual
+  allow_failure: false
```

Comment:
why so?

Reply:
Manual jobs are allowed to fail by default and don't block the following stages. By making it not allowed to fail, the next stage won't trigger until these jobs are green.